src/FakeItEasy.Net35/Compatibility/ConcurrentQueue.cs

```diff
@@ -0,0 +1,34 @@
+namespace FakeItEasy

```

Comment:
Just tried it, it works fine with a reference alias. See this commit: https://github.com/thomaslevesque/FakeItEasy/commit/ec736a8b87b2ff22d504f80e4dd1c4896d480a3d